internal/feeder/rekor/rekor_feeder.go

```diff
@@ -65,14 +80,24 @@ func FeedLog(ctx context.Context, l config.Log, w feeder.Witness, c *http.Client
 		if err := getJSON(ctx, c, lURL, "api/v1/log", &li); err != nil {
 			return nil, fmt.Errorf("failed to fetch log info: %v", err)
 		}
-		return []byte(li.SignedTreeHead), nil
+		// Active shard
+		if li.TreeID == l.ID {
+			return []byte(li.SignedTreeHead), nil
+		}
+		// Search inactive shards
+		for _, shard := range li.InactiveShards {
+			if shard.TreeID == l.ID {
+				return []byte(shard.SignedTreeHead), nil
+			}
+		}
+		return nil, errors.New("failed to find shard that matched log ID from config")
```

Comment:
super nit: I'd be tempted to do the `fmt.Errorf(... l.ID)` thing here; seeing what's being looked for could help take one indirection off trying debug an issue when it's not working.